rules/coverage_transitions.bzl

```diff
@@ -0,0 +1,60 @@
+"""Transition helpers used to build tools without coverage instrumentation.
```

Comment:
optional: should we restrict visibility to just this repo: https://bazel.build/rules/lib/globals/bzl#visibility